content/espanol/security-phishing/whats-phishy.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "¿Qué es phishy?"
+description: ""
+draft: false
+weight: 5
+---
+ 
+ 
+En nuestra última sección aprendimos sobre diferentes tipos de métodos de phishing y cómo reconocerlos. Ahora es el momento de probar tu conocimiento a través de un juego llamado  `¿Qué es phishy?`?
+
+¿Qué imágenes de la lista que se da a continuación crees que son phishy?
+
+![Correo electrónico de Spotify](../media/Spotify_email.jpg?classes=border,shadow)
```

Comment:
We will probably need to translate the Phishing image examples into Spanish as well